src/components/DlToastMessage/components/ToastComponent.vue

```diff
@@ -0,0 +1,195 @@
+<template>
+    <transition
+        :enter-active-class="transition.enter"
+        :leave-active-class="transition.leave"
+    >
+        <div
+            v-show="isActive"
+            ref="root"
+            class="v-toast__item"
+            :class="[`v-toast__item--${type}`, `v-toast__item--${position}`]"
+        >
+            <dl-alert
+                :type="type"
+                :closable="closable"
+                @update:model-value="(val) => closeToast(val)"
+            >
+                <!-- eslint-disable vue/no-v-html -->
+                <span
+                    :style="{ lineHeight: `${lineHeight}px` }"
+                    class="v-toast__text"
+                    data-test="message-text"
+                    v-html="message"
+                />
+                <!--eslint-enable-->
+            </dl-alert>
+        </div>
+    </transition>
+</template>
+
+<script lang="ts">
+import {
+    defineComponent,
+    ref,
+    onBeforeMount,
+    computed,
+    onMounted
+} from 'vue-demi'
+import DlAlert from '../../DlAlert.vue'
+import { Positions, Types } from '../helpers/config'
+import { removeElement } from '../helpers/render'
+import { Animation } from '../types'
+
+export default defineComponent({
+    name: 'ToastComponent',
+    components: { DlAlert },
+    props: {
+        message: {
+            type: String,
+            default: ''
+        },
+        type: {
+            type: String,
+            default: 'success',
+            validator(value: string): boolean {
+                return Object.values(Types as unknown).includes(value)
+            }
+        },
+        duration: {
+            type: Number,
+            default: 10
+        },
+        position: {
+            type: String,
+            default: Positions.bottom,
+            validator(value: string): boolean {
+                return Object.values(Positions as unknown).includes(value)
+            }
+        },
+        activeDuration: {
```

Comment:
Merge both duration and active Duration into single prop
with a default of 5 seconds in ms so 5000

the type of the prop is either number | null where if we pass null it will not have any duration

Reply:
the task says 10 seconds by default

Reply:
Ok 10 seconds then